src/executionTimingsReporter.ts

```diff
@@ -31,13 +41,13 @@ export class StepProgressEventsEmitter extends EventEmitter {
         this.emit(milestoneReachedEventName, { milestoneName: milestoneName } as MilestoneReachedEventArguments);
     }
 
-    private subscribeToAllNestedEmitters(event: string | symbol, listener: Function): void {
+    private subscribeToAllNestedEmitters(event: string, listener: Function): void {
         for (const nestedEventEmitter of this._nestedEmitters) {
-            nestedEventEmitter.on(event, listener);
+            nestedEventEmitter.on(event as any, listener as any);
```

Comment:
Wouldn't an 'event` parameter be just a string?

Reply:
the parameter is something like 'stepStarted'|'milestoneReached'|'navigatedToUserRequestedUrl'